pkgs/tools/networking/hp2p/default.nix

```
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, openmpi, python2Packages, autoconf, automake } :
```

```suggestion
{ stdenv, fetchFromGitHub, mpi, python2Packages, autoconf, automake } :
```
See https://github.com/NixOS/nixpkgs/pull/108983.